met-web/tests/unit/components/polyfills/cssEscape.ts

```diff
@@ -0,0 +1,23 @@
+import cssEscape from 'css.escape';
+
+/**
+ * jsdom does not ship CSS.escape, but MUI relies on it when building selectors around generated IDs.
+ * Provide the browser polyfill so selector queries stay valid during tests.
+ */
+type CssNamespace = {
+    escape?: (value: string) => string;
+};
+
+const ensureCssEscape = () => {
+    const globalObject =
+        typeof globalThis.window === 'undefined' ? (globalThis as Window & typeof globalThis) : globalThis.window;
```

Comment:
It looks like SonarQubeCloud suggests comparing directly here, instead of using typeof.